### ISSUE TITLE
Impossible to customize

### ISSUE THREAD

**Issue by Benjy763** [ui-jar/ui-jar#68]
Hello,
First of all congrats for the work and initiative for such an important tools like that.
I'm trying to customize the style and titles. I followed the readme but I don't know how it can work actually.
For the style, I imported the default file from dist but there is no variables inside ! So obviously it can't work.
You have the variable in dev mode in your repo but when all is built the generated css does not contain vars anymore.
Concerning the UIJarConfiguration, it does not exist into the app module, it does not exist in the dist at all. So it can't work too.
I did not find any issue on the subject, however it is an important feature.

Thanks

**Comment by nording** [comment]
Hi,
The readme on github is what is available in master. The new "UIJarConfiguration" object will be available in next release. Check out how you can customize current version at npm - https://www.npmjs.com/package/ui-jar

**Comment by Benjy763** [comment]
Thanks for answer, indeed it was the readme from master...The next release will be cool :)
I looked on 1.1.8 and there is no clean solution about the customisation today...?

**Comment by nording** [comment]
Yes, next release is going to allow more styling :) In 1.1.8 there is only possible to override CSS styling.
